junction/conferences/models.py

```diff
@@ -78,6 +78,7 @@ class ConferenceProposalReviewer(AuditModel):
     conference = models.ForeignKey(Conference, related_name='proposal_reviewers')
     reviewer = models.ForeignKey(User)
     active = models.BooleanField(default=True, verbose_name="Is Active?")
+    nick = models.CharField(max_length=255, verbose_name="Nick Name", null=True, blank=True)
```

Comment:
Fixed.

-- 
Cheers,

dhilipsiva

Full Stack & DevOps Engineer

http://dhilipsiva.com

+91 81 97 98 53 97

    


On 23 June 2015 at 09:15:40, Kracekumar Ramaraju (notifications@github.com) wrote:

In junction/conferences/models.py:

> @@ -78,6 +78,7 @@ class ConferenceProposalReviewer(AuditModel):
>      conference = models.ForeignKey(Conference, related_name='proposal_reviewers')
>      reviewer = models.ForeignKey(User)
>      active = models.BooleanField(default=True, verbose_name="Is Active?")
> +    nick = models.CharField(max_length=255, verbose_name="Nick Name", null=True, blank=True)
can we have default value as "Reviewer" ?

—
Reply to this email directly or view it on GitHub.